data/dconf/ibus.convert

```diff
@@ -16,10 +16,12 @@ trigger = /desktop/ibus/general/trigger
 
 [org.freedesktop.ibus.panel]
 custom-font = /desktop/ibus/general/custom_font
+custom-theme = /desktop/ibus/general/custom_theme
 lookup-table-orientation = /desktop/ibus/general/lookup_table_orientation
```

Comment:
I will deprecate dconf settings because the supported formats are different between dconf and gsettings so please drop the changes of ibus.convert.